samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -247,6 +247,13 @@ Test not discoverable advertising by completing `Testing`_ and the following add
 #. Press **Button 2** to increase the audio volume.
 #. Press **Button 4** to decrease the audio volume.
 
+Battery Notification extension
+------------------------------
+
+To test `Fast Pair Battery Notification extension`_ you have to be paired with at least one Fast Pair Seeker and put Fast Pair Provider in not discoverable advertising mode.
+Currently, Android phones have trouble with the Battery Notification extension and sometimes do not display battery information.
```

Comment:
Please double check with Fast Pair Provider type that is suggested by Google. If it works, we may need to update the documentation to ensure that user is aware of additional requirements.